### PR TITLE
Replaced deprecated kCCAlgorithmAES128 with kCCAlgorithmAES

### LINKED ISSUE #1199: Deprecation: Replace usages of deprecated kCCAlgorithmAES128 with kCCAlgorithmAES
In `CommonCryptor.h`,
```
enum {
    kCCAlgorithmAES128 = 0, /* Deprecated, name phased out due to ambiguity with key size */
    kCCAlgorithmAES = 0,
    kCCAlgorithmDES,
    kCCAlgorithm3DES,
    kCCAlgorithmCAST,
    kCCAlgorithmRC4,
    kCCAlgorithmRC2,
    kCCAlgorithmBlowfish
};
```

I believe we should replace the 2 usages of `kCCAlgorithmAES128` with `kCCAlgorithmAES`.

┆Issue is synchronized with this [Jira Uncategorised](https://ably.atlassian.net/browse/SDK-1331) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by maratal**
Closes #1199

### CODE REVIEW

**Review by lukasz-szyszkowski — APPROVED** [review]

File: Source/ARTCrypto.m [hunk 1]
```diff
@@ -255,7 +255,7 @@ - (ARTStatus *)decrypt:(NSData *)ciphertext output:(NSData *__autoreleasing *)ou
 - (NSString *)cipherName {
     NSString *algo = nil;
     switch (self.algorithm) {
-        case kCCAlgorithmAES128:
+        case kCCAlgorithmAES:
             algo = @"aes-128";
```
Comment (ben-xD) [inline Source/ARTCrypto.m:259]: Can we also update the cipherName to be consistent? e.g. `algo = @"aes"`?

**Review by ben-xD — CHANGES_REQUESTED** [review]

**Comment by maratal** [comment]
All greens in two consecutive attempts, @ben-xD

File: Source/ARTCrypto.m (hunk 1 above)
Comment (maratal) [reply Source/ARTCrypto.m]: 8364a0e

**Review by ben-xD — APPROVED** [review]

File: Source/ARTCrypto.m (hunk 1 above)
Comment (lukasz-szyszkowski) [reply Source/ARTCrypto.m]: @ben-xD please resolve this conversation if everything is ok. Thanks

**Pull request merged**